File: ironman/src/main/java/com/avengers/ironman/utils/LogUtils.java
package com.avengers.ironman.utils;

import android.util.Log;

public class LogUtils {
    private static final String TAG = "IronMan";

    public static void info(String msg) {
        Log.i(TAG, msg);
    }

    public static void debug(String msg) {
        Log.d(TAG, msg);
    }

    public static void error(String msg) {
        Log.e(TAG, msg);
    }
}

File: ironman/src/main/java/com/avengers/ironman/utils/LogUtils.kt
package com.avengers.ironman.utils

import android.util.Log

object LogUtils {

    private const val TAG = "IronMan"

    fun info(msg: String) {
        Log.i(TAG, msg)
    }

    fun debug(msg: String) {
        Log.d(TAG, msg)
    }

    fun error(msg: String) {
        Log.e(TAG, msg)
    }
}